client-react/src/pages/app/functions/function/function-log/FunctionLogAppInsightsDataLoader.tsx

```diff
@@ -323,6 +331,7 @@ const FunctionLogAppInsightsDataLoader: React.FC<FunctionLogAppInsightsDataLoade
       loadingMessage={loadingMessage}
       appInsightsResourceId={appInsightsComponent ? appInsightsComponent.id : ''}
       showFilteredLogsMessage={showFilteredLogsMessage}
+      useNewFunctionLogsApi={useNewFunctionLogsApi()}
```

Comment:
If using `useMemo` instead of `useCallback`:
```suggestion
      useNewFunctionLogsApi={useNewFunctionLogsApi}
```